vendor/bat-native-ledger/src/bat/ledger/internal/credentials/credentials_common.cc

```diff
@@ -134,14 +134,34 @@ void CredentialsCommon::GetSignedCredsFromResponse(
     return;
   }
 
+  auto* signed_creds = parsed_response.FindListKey("signed_creds");
+  if (!signed_creds || signed_creds->GetList().empty()) {
+    BLOG(ledger_, ledger::LogLevel::LOG_ERROR) <<
+        "Failed to parse signed creds";
+    callback(ledger::Result::RETRY);
```

Comment:
We need to exponentially backoff server retries, as otherwise if there is a genuine server issue that does not resolve itself we will use up a users data plan

Reply:
https://github.com/brave/brave-browser/issues/9685